src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -11,11 +11,13 @@
          This is probably because the code coverage target invocation does not canonicalize the
          process name. -->
     <ToolHost Condition="'$(OS)' == 'Windows_NT'">CoreRun.exe</ToolHost>
-    <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
```

Comment:
I am actually confirming that this is right, since I was seeing that the right name was with caps on my Linux machine but it doesn't look that way on the TestHost nuget package, so I'll make sure that this is correct, or revert this line if it's no longer correct.